### PR TITLE
Add China to the list of events

### PR DESCRIPTION
**Pull request by gerl1ng**
KubeCon + CloudNativeCon is now held thrice a year.
China is added to the list of events.

Signed-off-by: Marius Gerling <marius@gerl1ng.de>

### CODE REVIEW

File: contributors/guide/README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ If you haven't noticed by now, we have a large, lively, and friendly open-source
 
 ## Events
 
-Kubernetes is the main focus of CloudNativeCon/KubeCon, held twice per year in EMEA and in North America. Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
+Kubernetes is the main focus of KubeCon + CloudNativeCon, held thrice per year in China, EMEA and in North America. Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
```
Comment (cblecker) [inline contributors/guide/README.md]: I'd suggest "held three times per year" over "held thrice per year".

Comment (gerl1ng) [reply contributors/guide/README.md]: Should EMEA also be replaced by Europe as the conferences are called KC+CNC China / Europe / North America?

Comment (gerl1ng) [reply contributors/guide/README.md]: Changed and squashed it.

**Pull request merged**